CMakeLists.txt

```diff
@@ -712,8 +712,8 @@ else()
 endif()
 
 # Boost
-find_public_dependency(Boost REQUIRED COMPONENTS system)
-target_include_directories(torrent-rasterbar PUBLIC ${Boost_INCLUDE_DIRS})
+find_package(Boost REQUIRED COMPONENTS system)
+target_include_directories(torrent-rasterbar PUBLIC ${Boost_INCLUDE_DIR})
```

Comment:
I couldn't get this to work right. For some reason both `find_public_dependency` and `find_package` would find the system installed boost library but headers from some boost repository somewhere in my home directory. setting clearing `BOOST_ROOT` did not help. @zeule do you have any ideas of how to fix this? It results in a serious link error that's hard to diagnose.

Reply:
I suppose you tried cleaning the cmake cache in the build dir? Where does the `Boost_DIR` variable point to?

Reply:
Forgot to ask: which Boost installation you want the build to use?

Reply:
either is fine. The problem is that the library and the include path point to different ones, and then cause link errors

Reply:
If you set `Boost_DIR`, the `FindBoost.cmake` module should not go outside of it, IIRC. So, could you try to set it and run cmake again? You have to clean cmake cache (or use a new build directory, obviously simpler for testing).